addons/website_blog/static/src/js/website_blog.js

```diff
@@ -1,84 +1,137 @@
-        }
-    }
-    window.open(url, "", "menubar=no, width=500, height=400");
-}
+sAnimations.registry.websiteBlog = sAnimations.Class.extend({
+    selector: '.website_blog',
+    read_events: {
+        'click .cover_footer': '_pageTransist',
+        'click a[href^="#blog_content"]': '_animate',
+        'click .o_twitter': '_shareArticle',
+        'click .o_facebook': '_shareArticle',
+        'click .o_linkedin': '_shareArticle',
+        'click .o_google': '_shareArticle',
+        'click .o_twitter_complete': '_shareArticle',
+        'click .o_facebook_complete': '_shareArticle',
+        'click .o_linkedin_complete': '_shareArticle',
+        'click .o_google_complete': '_shareArticle',
+        'click .blog_post_year_collapse': '_toggleChevron',
+        'mouseenter div.o_blog_post_complete a': '_hoverBlogPost',
+        'mouseleave div.o_blog_post_complete a': '_unhoverBlogPost'
+    },
 
-$(document).ready(function() {
-    if ($('.website_blog').length) {
-        $(".js_tweet, .js_comment").share({});
-        $('.cover_footer').on('click', page_transist);
-        $('a[href^="#blog_content"]').on('click', animate);
-        $('.o_twitter, .o_facebook, .o_linkedin, .o_google, .o_twitter_complete, .o_facebook_complete, .o_linkedin_complete, .o_google_complete').on('click', shareArticle);
-        /* Archives years collapse */
-        $('.blog_post_year_collapse').on('click', function() {
-            $(this).find('i.fa').toggleClass('fa-chevron-down fa-chevron-right');
-        });
+    /**
+     * @override
+     */
+    start: function () {
         /* Collapse in active year */
         var $activeYear = $('.blog_post_year li.active');
         if ($activeYear.length) {
             var id = $activeYear.closest('ul').attr('id');
             $("li.blog_post_year_collapse[data-target='#"+ id +"']").click();
         }
-    }
+    },
 
-    /* Sharing links hover in blogpost */
-    $('div.o_blog_post_complete a').hover(
-        function() { $('div.o_blog_post_complete a').not('#'+this.id).addClass('unhover'); },
-        function() { $('div.o_blog_post_complete a').not('#'+this.id).removeClass('unhover'); }
-    );
-});
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {Event} event
+     */
+    _pageTransist: function(event) {
+        event.preventDefault();
+        var newLocation = $('.js_next')[0].href;
+        var top = $('.cover_footer').offset().top;
+        $('.cover_footer').animate({
+            height: $(window).height()+'px'
+        }, 300);
+        $('html, body').animate({
+            scrollTop: top
+        }, 300, 'swing', function() {
+           window.location.href = newLocation;
+        });
+    },
+
+    /**
+     * @private
+     * @param {Event} event
+     */
+    _animate: function(event) {
```

Comment:
Name the event variables `ev` (this is not only a convention, this is important because the mighty browser "Chrome" chose to have a global variable named `event` at all times, so this may lead to tricky bugs).